Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracks_cff.py

```diff
@@ -0,0 +1,105 @@
+
+ctppsDiamondRawToDigiAlCaRecoProducer  = _ctppsDiamondRawToDigi.clone(rawDataTag = cms.InputTag('hltPPSCalibrationRaw'))
+totemTimingRawToDigiAlCaRecoProducer = _totemTimingRawToDigi.clone(rawDataTag = cms.InputTag('hltPPSCalibrationRaw') )
+ctppsPixelDigisAlCaRecoProducer = _ctppsPixelDigis.clone(inputLabel = cms.InputTag('hltPPSCalibrationRaw'))
+
+ctppsRawToDigiTaskAlCaRecoProducer = cms.Task(
+  ctppsDiamondRawToDigiAlCaRecoProducer,
+  totemTimingRawToDigiAlCaRecoProducer,
+  ctppsPixelDigisAlCaRecoProducer
+)
+
+ALCARECOPPSCalMaxTracksRaw2Digi = cms.Sequence(ctppsRawToDigiTaskAlCaRecoProducer)
+
+# 3 a). RECO - PIXELS
+#------------------------------------------------------
+
+from RecoPPS.Configuration.recoCTPPS_cff import *
+from RecoPPS.Local.ctppsPixelClusters_cfi import ctppsPixelClusters as _ctppsPixelClusters
+from RecoPPS.Local.ctppsPixelRecHits_cfi import ctppsPixelRecHits as _ctppsPixelRecHits
+from RecoPPS.Local.ctppsPixelLocalTracks_cfi import ctppsPixelLocalTracks as _ctppsPixelLocalTracks
+
+ctppsPixelClustersAlCaRecoProducer = _ctppsPixelClusters.clone(tag = 'ctppsPixelDigisAlCaRecoProducer')
+ctppsPixelRecHitsAlCaRecoProducer = _ctppsPixelRecHits.clone(RPixClusterTag = 'ctppsPixelClustersAlCaRecoProducer')
+ctppsPixelLocalTracksAlCaRecoProducer = _ctppsPixelLocalTracks.clone(tag = 'ctppsPixelRecHitsAlCaRecoProducer')
+
+ctppsPixelLocalReconstructionTaskAlCaRecoProducer = cms.Task(
+    ctppsPixelClustersAlCaRecoProducer,
+    ctppsPixelRecHitsAlCaRecoProducer,
+    ctppsPixelLocalTracksAlCaRecoProducer
+)
+
+# 3 b). RECO - CTPPS DIAMONDS
+#------------------------------------------------------
+
+from RecoPPS.Local.ctppsDiamondRecHits_cfi import ctppsDiamondRecHits as _ctppsDiamondRecHits
+from RecoPPS.Local.ctppsDiamondLocalTracks_cfi import ctppsDiamondLocalTracks as _ctppsDiamondLocalTracks
+ctppsDiamondRecHitsAlCaRecoProducer = _ctppsDiamondRecHits.clone(digiTag = cms.InputTag('ctppsDiamondRawToDigiAlCaRecoProducer','TimingDiamond'))
+ctppsDiamondLocalTracksAlCaRecoProducer = _ctppsDiamondLocalTracks.clone(recHitsTag = 'ctppsDiamondRecHitsAlCaRecoProducer')
+
+ctppsDiamondLocalReconstructionTaskAlCaRecoProducer = cms.Task(
+    ctppsDiamondRecHitsAlCaRecoProducer,
+    ctppsDiamondLocalTracksAlCaRecoProducer
+)
+
+# 3 c). RECO - TIMING DIAMONDS
+#------------------------------------------------------
+
+from RecoPPS.Local.totemTimingRecHits_cfi import totemTimingRecHits as _totemTimingRecHits
+from RecoPPS.Local.diamondSampicLocalTracks_cfi import diamondSampicLocalTracks as _diamondSampicLocalTracks
+
+totemTimingRecHitsAlCaRecoProducer = _totemTimingRecHits.clone(digiTag = cms.InputTag('totemTimingRawToDigiAlCaRecoProducer','TotemTiming'))
+diamondSampicLocalTracksAlCaRecoProducer = _diamondSampicLocalTracks.clone(recHitsTag = 'totemTimingRecHitsAlCaRecoProducer')
+
+diamondSampicLocalReconstructionTaskAlCaRecoProducer = cms.Task(
+		totemTimingRecHitsAlCaRecoProducer,
```

Comment:
to me this looks indented, I hope that's fine for python, maybe it would be better to fix the indentation anyway?

Reply:
fixed